jobs/openshift/cluster/create/Jenkinsfile

```diff
@@ -48,6 +48,30 @@ node('cirhos_rhel7') {
         if (params.dryRun) {
             println("Would call ansibleTower with: oo_clusterid: ${clusterOptions.name}, oo_sublocation: ${clusterOptions.awsRegion}")
         } else {
+            def extraVars = [
+                "oo_clusterid: ${clusterOptions.name}",
+                "oo_sublocation: ${clusterOptions.awsRegion}",
+                "cluster_credential_bundle_aws_name: ${clusterOptions.awsAccountName}",
+                "survey_provision_logging: 'false'",
+                "master_instance_size: ${clusterOptions.nodes.master.instance_type}",
+                "compute_instance_size: ${clusterOptions.nodes.compute.instance_type}",
+                "infra_instance_size: ${clusterOptions.nodes.infra.instance_type}",
+                "openshift_aws_master_group_desired_size: ${clusterOptions.nodes.master.group_size}",
+                "openshift_aws_compute_group_desired_size: ${clusterOptions.nodes.compute.group_size}",
+                "openshift_aws_infra_group_desired_size: ${clusterOptions.nodes.infra.group_size}"
+            ]
+            
+            if (clusterOptions.nodes.compute.group_size == '1') {
+                extraVars << 'openshift_aws_compute_group_min_size: 1'
+            }
+
+            if (clusterOptions.nodes.infra.group_size == '1') {
+                extraVars << 'openshift_aws_infra_group_min_size: 1'
+            }
```

Comment:
@odra Does setting these to 1 work, or is it always at least 2?

Reply:
if it is one it will set to one (it's not working atm) otherwise it will use the  default value